receiver/envoyalsreceiver/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+type: envoyals
+
+status:
+  class: receiver
+  stability:
+    alpha: [logs]
+  distributions: [contrib, k8s]
+  codeowners:
+    active: [evan-bradley]
```

Comment:
don't you want to be a codeowner as well? Are you an OpenTelemetry member?

Reply:
I'm happy to be a codeowner, but right now I'm not an OpenTelemetry member.

Reply:
Please discuss with your component sponsor. Please see to open an issue under https://github.com/open-telemetry/community/issues/new?template=membership.md

Reply:
I read it earlier and I will update my id here after I become a member.

Reply:
I'm okay being the sole codeowner on this until we've finished the initial set of PRs and added @zirain to the org so we can avoid the allowlist dance. I'll make sure we have proper ownership of this component.